### ISSUE TITLE
Need localStorage to be able to expire?

### ISSUE THREAD

**Issue by xiaoyuze88** [grevory/angular-local-storage#303]
In my cases, I use localStorage as temporary data cache.
  However, I need those data can be expired too. Now I will never know when those data is gonna expire unless I save the updating time in the data structure. So I think maybe these can be wrapped into the setter/getter functions?
   Thanks!

**Comment by chenjianjx** [comment]
Yes, I want it too.

**Comment by kamykm** [comment]
Yes. That is still needed. Additionally I wanted to use this instead of angular-cookies but unfortunately, I cannot set an exact expiration date for a cookie (days value is not good enough)
